raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -553,12 +562,19 @@ private boolean handleVoteResponse(
         }
     }
 
-    private int randomElectionJitterMs() {
-        if (electionJitterMs == 0)
+    private int randomElectionTimeoutMs() {
+        if (electionTimeoutMs == 0)
             return 0;
-        return random.nextInt(electionJitterMs);
+        // avoiding float calculations here
```

Comment:
nit: comment seems not needed?

Reply:
yeah I tried to emphasize we want to avoid (1 + random(1.0)) * timeout, but on a second thought it is not very crucial so I'm okay to remove this comment.